Changes to header-extended

kbchumley opened the pull request:
- adding conditional statements
-changing .md nav text

(needs review @pglevy - Having issues with drop down orientation and am unsure if I need to condense top information on us government statement along with translate, contact and subscribe)

kbchumley committed:
Changes to header-extended

- adding conditional statements
-changing .md nav text

pglevy commented:
Looking good! Not sure what you mean by "drop down orientation," but if you mean that the buttons are opening the wrong menus, it's because we have the accordion parameters hard-coded. They need to match. One approach is to use the built-in template `{{ forloop.index }}`, which gives you a number based on the item the loop is currently on. That way, the button and the list id will always match.

For the spacing, there's some padding on the body tag in default.html.

![image](https://user-images.githubusercontent.com/5177337/127365704-93a6e76b-17f9-47d1-9665-21053dad85df.png)

kbchumley committed:
forloop.index

- still having issues with drop down placement, i tried this as well but nothing {% if forloop.index == 1 %}
